pkg/transformers/json/transformer.go

```diff
@@ -70,6 +90,13 @@ func transformer(msg messaging.Message) (interface{}, error) {
 				return nil, errors.Wrap(ErrTransform, errInvalidNestedJSON)
 			}
 			newMsg := ret
+
+			// Apply timestamp transformation rules depending on key/unit pairs
+			ts, err := ts.transformTimeField(v)
+			if ts != 0 && err != nil {
+				newMsg.Created = ts
+			}
+
```

Comment:
What if error has happened? Would it make sense to log some output? Or we just do not set timestamp and that's it?